grakn-graph/src/test/java/ai/grakn/graph/internal/GraknGraphTest.java

```diff
@@ -408,117 +396,4 @@ public void whenShardingSuperNode_EnsureNewInstancesGoToNewShard(){
-                "category sub entity\n" +
-                "    has name\n" +
-                "    plays subcategory\n" +
-                "    plays supercategory\n" +
-                "    plays 'label'\n" +
-                "    plays item\n" +
-                "    plays recommended;\n" +
-                "name sub resource datatype string;\n" +
-                "hierarchy sub relation\n" +
-                "    relates subcategory\n" +
-                "    relates supercategory;\n" +
-                "category-assignment sub relation\n" +
-                "    has rank\n" +
-                "    relates item #product\n" +
-                "    relates 'label'; #category \n" +
-                "rank sub resource datatype long;\n" +
-                "user sub entity\n" +
-                "    has uid\n" +
-                "    has username\n" +
-                "    plays reviewer\n" +
-                "    plays buyer;\n" +
-                "uid sub ID;\n" +
-                "username sub name;\n" +
-                "completed-recommendation sub relation\n" +
-                "    relates successful-recommendation\n" +
-                "    relates buyer;\n" +
-                "implied-recommendation sub relation\n" +
-                "    relates category-recommendation\n" +
-                "    relates product-recommendation;\n" +
-                "recommendation sub relation is-abstract\n" +
-                "    plays successful-recommendation\n" +
-                "    plays product-recommendation;\n" +
-                "co-categories sub relation\n" +
-                "    plays category-recommendation\n" +
-                "    relates item\n" +
-                "    relates recommended;\n" +
-                "also-viewed sub recommendation\n" +
-                "    relates item\n" +
-                "    relates recommended;\n" +
-                "also-bought sub recommendation\n" +
-                "    relates item\n" +
-                "    relates recommended;\n" +
-                "bought-together sub recommendation\n" +
-                "    relates item\n" +
-                "    relates recommended;\n" +
-                "transaction sub relation\n" +
-                "    relates buyer\n" +
-                "    relates item;\n" +
-                "asked-question sub role;\n" +
-                "given-answer sub role;\n" +
-                "item sub role;\n" +
-                "feedback sub role;\n" +
-                "reviewer sub role;\n" +
-                "buyer sub role;\n" +
-                "recommended sub role;\n" +
-                "subcategory sub role;\n" +
-                "supercategory sub role;\n" +
-                "'label' sub role;\n" +
-                "successful-recommendation sub role;\n" +
-                "category-recommendation sub role;\n" +
-                "product-recommendation sub role;").execute();
-        graknGraph.commit();
-    }
```

Comment:
We could move this to `grakn-graql`. I don't really have a problem with deleting it though, since I don't know what this test is for.